conf/application.conf

```diff
@@ -93,6 +93,11 @@ mongodb {
   uri = "mongodb://localhost:27017/eu-subsidy-compliance"
 }
 
+email {
```

Comment:
might be good to add something to make it obvious that these are templateIds. Maybe rename it or add a comment

Reply:
I agree, have added a comment